tests/functional/ext/docstyle/docstyle_quotes_bad.py

```diff
@@ -0,0 +1,18 @@
+"""Checks of Dosctrings 'bad-docstring-quotes'"""
+# pylint: disable=docstring-first-line-empty,missing-class-docstring
+
+
+class FFFF:
```

Comment:
Could you merge the `bad` and `ok` files? That makes it a bit easier to see what this checker is checking specifically.

Reply:
Pierre had another opinion on this if I understood it right: https://github.com/PyCQA/pylint/pull/5526#issuecomment-993667652

Reply:
Ah I see what he means. I'll push a commit to your branch to do so. Just give me a couple of minutes :)

Reply:
thanks!